canonical/loki-k8s-operator
# Implement TLS and enable it when a cert is available

## Issue


This PR implements TLS as part of this tracking issue https://github.com/canonical/cos-lite-bundle/issues/75

## How to test it

1. Deploy Loki
2. Deploy `self-signed-certificates`
3. Deploy `prometheus-k8s`
4. Relate Loki to `self-signed-certificates` and `prometheus-k8s`
5. You can check HTTP endpoint in Prometheus:
![imagen](https://github.com/canonical/loki-k8s-operator/assets/939888/df73c023-b451-4dd6-94e9-898f950b37b9)
